pkg/tinyvcdiff/contrib/tinyvcdiff_vfs/vfs.c

```diff
@@ -0,0 +1,66 @@
+#include "vcdiff_vfs.h"
+#include "errno.h"
+
+static int read (void *dev, uint8_t *dest, size_t offset, size_t len)
```

Comment:
```suggestion
static int _read(void *dev, uint8_t *dest, size_t offset, size_t len)
```

Now you *really* don't want to clash with [`read(2)`](https://man7.org/linux/man-pages/man2/read.2.html)